services/comprehension/feedback-api-main/src/endpoint/endpoint_test.go

```diff
@@ -226,6 +226,7 @@ func TestIdentifyUsedFeedbackIndex(t *testing.T) {
 	feedbacks[1] = optimal_response
 	feedbacks[2] = usable_used_response
 	feedbacks[3] = usable_unused_response
+	feedbacks[4] = usable_unused_response
```

Comment:
What's the reason for storing this twice?

Reply:
By convention, golang returns the default automl response if everything else is optimal. And the existing code makes assumptions about the relative precedence of the automl API. Thus, we need to this massaging.